tests/integration/runLinter.js

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+import { exec } from 'child_process';
+
+
+export const BIN_PATH = path.join(__dirname, '../../bin/');
+export function runLinter(args = [], options = {}) {
```

Comment:
it would be better to make this test helper a bit more generic, just enough to be able to use it run also the other bin scripts (so that we can use the integration test suite also for the bin scripts that are used to update the locales and the Firefox API schema files), and then also rename the module name (e.g. maybe it could be named `helpers.js` instead of `runLinter.js`, like the one used by the unit tests).